plenum/persistence/req_id_to_txn.py

```diff
@@ -13,33 +13,41 @@ class ReqIdrToTxn:
     def __init__(self, keyValueStorage: KeyValueStorage):
         self._keyValueStorage = keyValueStorage
 
-    def add(self, digest, ledger_id, seq_no):
-        self._keyValueStorage.put(digest, self._create_value(ledger_id, seq_no))
-
-    def addBatch(self, batch):
-        self._keyValueStorage.setBatch([(digest, self._create_value(ledger_id,
-                                                                    seq_no))
-                                        for digest, ledger_id, seq_no in batch])
-
-    def get(self, digest):
+    def add(self, payload_digest, ledger_id, seq_no, digest):
+        self._keyValueStorage.put(payload_digest, self._create_value(ledger_id, seq_no, digest))
```

Comment:
I think there is no need to store `digest` as a value for `payload_digest` key, isn't it?

Reply:
yes